.scripts/build.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 
```

Comment:
Should we delete this file since its not being used, or do we still plan to move the build steps here?

Reply:
I will make a note. Actually previously in this PR: https://github.com/numpy/numpy.org/pull/46 , I used to call the `build.sh` and `deploy.sh` separately. But now we were facing [issue](https://github.com/numpy/numpy.org/pull/58#issuecomment-544115089) so I make it in one script file.

Reply:
Got it! I did see your comment in deploy.sh after I made this comment. Maybe we just delete or add a `#deprecated / not being used` comment in build.sh?